libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart-configuration.service.ts

```diff
@@ -83,6 +83,15 @@ export class FhirChartConfigurationService {
     });
   }
 
+  setSummaryRange(months: number) {
+    this.annotationSubject.next([this.buildTimeframeAnnotation('today', 0)]);
+    this.summaryRangeSubject.next({
+      months,
+      max: new Date().getTime(),
+      min: subtractMonths(new Date(), months).getTime(),
+    });
+  }
```

Comment:
Why is this method still emitting `summaryRangeSubject` if it is not adding the summary range annotations? Seems like the annotations should always be in sync with the summaryRange$ observable.